internal/rbd/diskusage.go

```diff
@@ -24,6 +24,15 @@ import (
 // rbd_sparify() to free zero-filled blocks and reduce the storage consumption
 // of the image.
 func (ri *rbdImage) Sparsify() error {
+	inUse, err := ri.isInUse()
```

Comment:
It would be good to document this restriction in the description of the function. Maybe include the reasoning why in-use images should not get `rbd sparsify`.

Reply:
Thanks!